perun-core/src/main/java/cz/metacentrum/perun/core/implApi/modules/attributes/AbstractMembershipExpirationRulesModule.java

```diff
@@ -35,6 +36,7 @@
 	public static final String membershipDoNotAllowLoaKeyName = "doNotAllowLoa";
 	public static final String autoExtensionLastLoginPeriod = "autoExtensionLastLoginPeriod";
 	public static final String autoExtensionExtSources = "autoExtensionExtSources";
+	public static final String doNotExpireSponsoredMembers = "doNotExpireSponsoredMembers";
```

Comment:
I would suggest changing the new value to a non-negated version. This makes it harder to understand when you have a `false` value.
```suggestion
	public static final String expireSponsoredMembers = "expireSponsoredMembers";
```

Reply:
It is also a possibility and I have no problem with it.

Reply:
OK, reworked.